### PR TITLE
Add my impression

### PR DESCRIPTION
**Pull request by pilgrimmeng**
This is my first attempt to pull request. Thank you for your guidance and support!

### CODE REVIEW

**Comment by liubin** [comment]
冲突了，请 rebase

**Comment by pilgrimmeng** [comment]
已经解决了@liubin

File: index.html [hunk 1]
```diff
@@ -75,6 +75,9 @@ <h3>读者来稿</h3>
 <select class="select"></select>
 
 <section class="wrap">
+
+
+<p class="impression" style="color: #2775b6">2021/8/17 能实践起来真的很有收获，感谢作者。(@hirocaster)</p>
```
Comment (liubin) [inline index.html:79]: 你确定 `hirocaster` 不改么？

Comment (pilgrimmeng) [reply index.html:79]: 我正在学习如何修改，谢谢您的提醒

**Pull request merged**